docs/content/en/docs/documentation/dependent-resource-and-workflows/dependent-resources.md

```diff
@@ -355,6 +355,42 @@ as [integration tests](https://github.com/operator-framework/java-operator-sdk/t
 To see how bulk dependent resources interact with workflow conditions, please refer to this
 [integration test](https://github.com/operator-framework/java-operator-sdk/tree/main/operator-framework/src/test/java/io/javaoperatorsdk/operator/dependent/bulkdependent/conidition).
 
+## Dependent Resources with External Resource
+
+Dependent resources are designed to manage also non-Kubernetes or external resources.
+To implement such dependent you can extend `AbstractExternalDependentResource` or one of it 
+[subclasses](https://github.com/operator-framework/java-operator-sdk/blob/main/operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/dependent/external).
+
+While with kubernetes resources we do some nice assumptions, like
```

Comment:
Starting with "While" but there is no follow up. Either a mistake or some following text was deleted.

Reply:
just needs a repharasing, follow up in the next section

Reply:
rephrased, thank you!